pkg/githubclient/client_repository.go

```diff
@@ -247,15 +248,34 @@ func (c *Client) getGithubChecks(ctx context.Context, repository *github.Reposit
 	var checks []string
 	for _, combinedStatus := range allCombinedStatus {
 		for _, status := range combinedStatus.Statuses {
-			checks = append(checks, *status.Context)
+			if filterCheck(status.GetContext()) {
+				checks = append(checks, status.GetContext())
+			}
 		}
 	}
 
 	c.logger.Debugf("found %d commit statuses for ref %q:\n%v", len(checks), ref, checks)
+	for id, check := range checks {
+		c.logger.Debugf(" - checks[%d] = %q", id, check)
+	}
 
 	return checks, nil
 }
 
+// Returns false if check should be ignored
```

Comment:
Correct Golang comment syntax for function is `// functionName comment`

```suggestion
// filterCheck returns false if check should be ignored
```